test-data/unit/check-inference.test

```diff
@@ -2174,8 +2174,8 @@ class A:
 [out]
 
 [case testMultipassAndTopLevelVariable]
-y = x # E: Cannot determine type of "x"  # E: Name "x" is used before definition
-y()
+y = x # E: Name "x" is used before definition
+y() # E: "int" not callable
```

Comment:
I think this might indicate a bug in mypy itself? I don't think this should change just from the constant folding